src/frontend/details/Details.js

```diff
@@ -1,19 +1,15 @@
 import React, { Component } from 'react';
-//import movieData from '../movie-data'
 import { Link, Redirect } from "react-router-dom";
 import './details.css'
-import noImage from '../no-image.svg'
+// import noImage from '../no-image.svg'
 
 export default class StatefulDetails extends Component {
   constructor(props) {
     super(props);
     this.state = {
       error: null,
       isLoaded: false,
-      title: '',
-      synopsis: 'Default Synopsis',
-      image: 'Default Image',
-      data: [{title: 'new movie'}]
+      shows: [],
```

Comment:
Mmm why are you saving the `shows`? I thought you just wanted to look after a single `show`

Reply:
amended